JSON_for_IO-Link.yaml

```diff
@@ -1101,13 +1114,13 @@ paths:
         '500':
            $ref: '#/components/responses/HTTP_500'
            
-  '/masters/{masterNumber}/configuration': #@todo wireles
+  '/masters/{masterNumber}/configuration':                                      #TODO: wireles
     get:
       operationId: GetMastersMasterNumberConfiguration
       tags:
         - masters
-      summary: Read the actual configuration of the specified Master.
-      description: Read the actual configuration of the specified Master.
+      summary: Read the actual configuration of the specified W-Master.
```

Comment:
I disagree on this change, since it applies also for wired masters

Reply:
For now or with the version I used IO-Link Wired has no Master configuration. Regarding of this I wrote W-Master to be clear that this configuration is only for Wireless.

Reply:
Endpoints should be as generic as possible and configuration handling is also a desirable feature for wired masters.  If a wired master has not implemented the configuration feature, this should be reflected in the return code.

Reply:
I Agree, change it back to Master

Reply:
You can edit this line by committing to your repo [SebastianHag:patch-1](https://github.com/SebastianHag/JSON_for_IO-Link/tree/patch-1) or pressing on the the three dots here https://github.com/iolinkcommunity/JSON_for_IO-Link/pull/68/files#top and then edit file